lisa/sut_orchestrator/baremetal/platform_.py

```diff
@@ -61,6 +71,18 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
         )
         self.cluster.initialize()
 
+    def _get_environment_information(self, environment: Environment) -> Dict[str, str]:
+        information: Dict[str, str] = {}
+        information[KEY_VMM_VERSION] = self._get_vmm_version(environment.default_node)
```

Comment:
Use a loop to visit `_environment_information_hooks`, so it's useful to extend in future.

Reply:
@squirrelsc  please check now